wallet/src/ui/app/redux/slices/txresults/index.ts

```diff
@@ -92,8 +107,19 @@ export const getTransactionsByAddress = createAsyncThunk<
                             }
                             const txn = txns[0];
                             const txKind = getTransactionKindName(txn);
+
+                            const tranferSui =
+                                txKind === 'TransferSui'
+                                    ? getTransferSuiTransaction(txn)
+                                    : null;
```

Comment:
```suggestion
                            const tranferSui = getTransferSuiTransaction(txn);
```

see the function definition: https://github.com/MystenLabs/sui/blob/6173c3a8ba0c3179ce800b448fb841f925195212/sdk/typescript/src/types/transactions.ts#L237